nipoppy/workflows/runner.py

```diff
@@ -153,6 +157,47 @@ def launch_boutiques_run(
 
         return descriptor_str, invocation_str
 
+    def _check_tar_conditions(self):
+        """
+        Make sure that conditions for tarring are met if tarring is requested.
+
+        Specifically, check that dpath to tar is specified in the tracker config
+        """
+        if not self.tar:
+            return
+
+        if self.pipeline_step_config.TRACKER_CONFIG_FILE is None:
+            raise RuntimeError(
+                "Tarring requested but is no tracker config file. "
+                "The TRACKER_CONFIG_FILE field needs to be specified in the "
+                "pipeline step config, and the PARTICIPANT_SESSION_DIR field "
+                "in that file must be specified"
+            )
+        if self.tracker_config.PARTICIPANT_SESSION_DIR is None:
+            raise RuntimeError(
+                "Tarring requested but no participant-session directory specified. "
+                "The PARTICIPANT_SESSION_DIR field in the tracker config must set "
```

Comment:
Ok, I named it like this because I thought this could be multi-purpose for derivatives discovery: if we add this path to the imaging bagel file then Neurobagel would know where to find the derivatives data (when building cohorts for example). But will ask around and see if there is a better name we could use